src/main/java/org/springframework/social/salesforce/api/SalesforceUserDetails.java

```diff
@@ -0,0 +1,50 @@
+package org.springframework.social.salesforce.api;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.io.Serializable;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SalesforceUserDetails implements Serializable {
```

Comment:
I know there are a few differences in property names and availability of data between the APIs, but they both do seem to represent the same user.

As such, can we either leverage the existing `SalesforceProfile` class or make `SalesforceProfile` an extension of `SalesforceUserDetails`?